index/models.py

```diff
@@ -46,7 +51,7 @@ class Signature(models.Model):
     signer = models.ForeignKey(
         User, on_delete=models.CASCADE, related_name="signatures"
     )
-    signed_date = models.DateTimeField(default=timezone.now())  # When they signed
+    signed_date = models.DateTimeField(default=date_default)  # When they signed
```

Comment:
So this fixes the problem, but you could also just do `default=timezone.now` and eliminate `date_default`